cookbooks/aws-parallelcluster-slurm/files/default/head_node_slurm/slurm/pcluster_fleet_config_generator.py

```diff
@@ -67,6 +67,7 @@ def generate_fleet_config_file(output_file, input_file):
 
                 if compute_resource_config.get("InstanceTypeList"):
                     fleet_config[queue][compute_resource] = {
+                        "API": "create-fleet",
```

Comment:
Probably in other places we have used `Api` and not `API`. Also need to update the method docs